app/ui/src/scss/utils/_overrides.scss

```diff
@@ -15,33 +15,31 @@ body.cards-pf {
   margin-bottom: 10px;
 }
 
-.layout-pf.layout-pf-fixed .container-pf-nav-pf-vertical.collapsed-nav {
-  margin-left: 0;
+.pf-c-page {
+  display: block;
+  grid-template-columns: none;
```

Comment:
Do you need these? If it isn't `display: grid;`, I would expect the `grid-template-*` properties to be ignored.

Reply:
By not adding `none` to `grid-template-columns` into the css you get these properties creep up:

<img width="570" alt="screen shot 2019-02-07 at 4 00 13 am" src="https://user-images.githubusercontent.com/1402829/52400404-08542b80-2a8d-11e9-9883-086a925160cc.png">

Reply:
Ah yeah, but those only apply to a `grid` parent. If you set `display: block;` on `.pf-c-page` (overriding `display: grid;`), those properties shouldn't do anything.